dashboard/app/controllers/levels_skills_controller.rb

```diff
@@ -27,6 +27,18 @@ def create
     end
   end
 
+  def delete
+    level_id = params[:level_id].to_i
+    skill_id = params[:skill_id].to_i
+    begin
+      level = Level.find(level_id)
+      Skill.find(skill_id).levels.destroy(level)
+      render json: {status: 'success', message: 'LevelsSkill deleted successfully'}, status: :ok
+    rescue ActiveRecord::RecordNotFound
+      render status: :not_found, json: "No LevelsSkill with level_id #{level_id} and skill_id #{skill_id}"
+    end
+  end
```

Comment:
this looks like it works, but it's a little unusual to be accessing the join table directly. a more conventional way to handle this would be to treat the list of skills as part of the level object, and either handle it in the existing LevelsController#update method, or have a separate method like LevelsController#update_skills, which receives a list of skill keys and sets level.skills = [...] appropriately. I would recommend that approach just to be on a more trodden path with less surprises. If you'd rather not do that or defer that until later, lemme know and I'll take a closer look at what you're doing here in this PR.

Reply:
I see what you mean. what you think about moving forward with this as is and then a follow-up refactor to move both the create and delete methods from this file into either the skills controller or the levels controller?

Reply:
per live discussion, I think it is fine to keep adding / removing individual levels skills via this controller, given that it matches your use case fairly closely.